types/correlation-id/tslint.json

```diff
@@ -0,0 +1,29 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
For new definitions do not add extra rules. Please fix this and use dts-gen to generate the package and do not modify tslint.json

Reply:
Thanks @sheetalkamat, as I mentioned in my MR, I felt that I had a need to use the `Function` type. The default tslint config provided by dts-gen completely forbids use of `Function`. Could you review my usage of `Function` at https://github.com/natemara/DefinitelyTyped/blob/c60e4b129485cd1fd8cbc073927eb45c5eaf733d/types/correlation-id/index.d.ts#L9 and make sure this is justified?

Reply:
you could write that as
```typescript
export function bindId<T extends (...p: any[]) => any>(id: string, work: T): T;
```